<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Animate</class>
 <widget class="QDialog" name="Animate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Animator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../_resourses/images.qrc">
    <normaloff>:/images/Icon16.png</normaloff>:/images/Icon16.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,1,80,10" columnstretch="80,0,0,0">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Direction</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="directLeft">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>Left</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="directRight">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Right</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QSpinBox" name="FrameSpeed">
     <property name="minimum">
      <number>32</number>
     </property>
     <property name="maximum">
      <number>1024</number>
     </property>
     <property name="value">
      <number>128</number>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>47</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="EditAnimationBtn">
     <property name="text">
      <string>Edit animations</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>106</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Frame speed:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QGraphicsView" name="AnimateView">
     <property name="minimumSize">
      <size>
       <width>256</width>
       <height>326</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="3" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Animation</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QListWidget" name="animationsList">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>directLeft</tabstop>
  <tabstop>directRight</tabstop>
  <tabstop>EditAnimationBtn</tabstop>
  <tabstop>FrameSpeed</tabstop>
  <tabstop>AnimateView</tabstop>
 </tabstops>
 <resources>
  <include location="../_resourses/images.qrc"/>
 </resources>
 <connections/>
</ui>
